### PR TITLE
Adding commands and jobs to ipa orb

### PR DESCRIPTION
**Pull request by simonarcher999**

### CODE REVIEW

File: ipa-deploy/orb.yml [hunk 1]
```diff
@@ -79,16 +91,217 @@ commands:
...
+          command: serverless plugin install -n serverless-plugin-aws-alerts
+          name: Install serverless-aws-alerts
+      - run:
+          command: |
+            export DEPLOYMENT_TYPE=integrationTest
+            serverless deploy --stage ${STAGE} --region ${AWS_REGION}
+          name: deploy
+      - run:
+          name: Checking env variable
+          command: echo ${KTR_REQUEST_INPUT_QUEUE}
```
Comment (gregshiel) [inline ipa-deploy/orb.yml:189]: specific only to KTR?

Comment (simonarcher999) [reply ipa-deploy/orb.yml:189]: oops was just using this for debugging will remove

File: ipa-deploy/README.md [hunk 2]
```diff
@@ -1,24 +1,67 @@
...
+
+### eslint
+
+This job is used to run eslint against the source code.
+
+### jest-tests
+
+This job is used to run the tests saved in the repo.
+
+### deploy
```
[the remark this answers is not in the document]
Comment (simonarcher999) [reply ipa-deploy/README.md:52]: I think it is in there, line 29

[the remark this answers is not in the document]
Comment (gregshiel) [reply ipa-deploy/README.md:52]: Ah wait was that always there above? sorry!

**Review by gregshiel — APPROVED** [review]

**Pull request merged**